src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -62,6 +68,7 @@ public class DatasetService implements ConsentLogger {
   private final StudyDAO studyDAO;
   private final DatasetServiceDAO datasetServiceDAO;
   private final UserDAO userDAO;
+  public Integer DATASET_BATCH_SIZE = 50;
```

Comment:
Since this isn't a static final, the [convention](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) is to use lowerCamelCase instead of SCREAMING_SNAKE_CASE.